src/cascade/executor/session_options.py

```diff
@@ -11,39 +11,38 @@ def make_options(ev_settings):
     options = dict()
     # parent location id will be set for us
     # parent node name is set for us
-    options["meas_std_effect"] = "add_std_scale_all"
-    if hasattr(ev_settings, "zero_sum_random") and ev_settings.zero_sum_random:
-        options["zero_sum_random"] = " ".join(ev_settings.zero_sum_random)
+    options["meas_std_effect"] = ev_settings.policies.meas_std_effect
+    if not ev_settings.model.is_field_unset("zero_sum_random"):
+        options["zero_sum_random"] = " ".join(ev_settings.model.zero_sum_random)
     # data extra columns is set for us.
     # avgint extra columns is set for us.
     options["warn_on_stderr"] = True
-    if hasattr(ev_settings, "minimum_meas_cv") and ev_settings.minimum_meas_cv:
-        options["minimum_meas_cv"] = ev_settings.minimum_meas_cv
-
     options["ode_step_size"] = ev_settings.model.ode_step_size
-    if ev_settings.model.additional_ode_steps:
+    if not ev_settings.model.is_field_unset("additional_ode_steps"):
         options["age_avg_split"] = " ".join(str(ea) for ea in ev_settings.model.additional_ode_steps)
-    if hasattr(ev_settings, "random_seed") and ev_settings.random_seed:
-        options["random_seed"] = ev_settings.random_seed
+    if not ev_settings.model.is_field_unset("random_seed"):
+        options["random_seed"] = ev_settings.model.random_seed
     # rate case is set for us.
 
     for kind in ["fixed", "random"]:
         for opt in ["derivative_test", "max_num_iter", "print_level", "accept_after_max_steps", "tolerance"]:
             if hasattr(ev_settings, opt):
```

Comment:
Another hasattr, though this one may actually be necessary if the fixed and random forms have different fields